### PR TITLE
added support for http_api_version

### PR DESCRIPTION
**Pull request by tarekziade**
See https://github.com/mozilla-services/cliquet/issues/598

### CODE REVIEW

**Comment by leplatrem** [comment]
* [ ] Add entry in changelog to describe what's going on :)

Update docs (if possible using `versionchanged::` and `versionadded::`)

* [ ] configuration.rst
* [ ] versioning.rst
* [ ] deprecation.rst
* [ ] utilities.rst

**Comment by tarekziade** [comment]
> versioning.rst

you said we need to remove it at https://github.com/Kinto/kinto/pull/300#discussion-diff-45957947

Do you want to keep it or remove it at this point ?

**Comment by tarekziade** [comment]
> configuration.rst & deprecation.rst

Could you expand on what to do there ?

File: cliquet_docs/api/utilities.rst [hunk 1]
```diff
@@ -8,12 +8,14 @@ GET /
...
 The returned value is a JSON mapping containing:
 
-- ``hello``: the name of the service (e.g. ``"reading list"``)
-- ``protocol_version``: the cliquet protocol version (``"2"``)
-- ``version``: complete application/project version (``"X.Y.Z"``)
+.. versionchanged:: 2.20::
+
+- ``project_name``: the name of the service (e.g. ``"reading list"``)
+- ``project_version``: complete application/project version (``"X.Y.Z"``)
+- ``project_documentation``: The URL to the service documentation. (this document!)
```
Comment (Natim) [inline cliquet_docs/api/utilities.rst]: The view implements ``project_docs``

**Comment by Natim** [comment]
r+

**Pull request merged**